constellation-comp-config/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "sourceMap": true,
     "module": "es2020",
```

Comment:
Config needs to be updated similar to root tsconfig

Reply:
> Config needs to be updated similar to root tsconfig

updated